docs/index.mdx

```diff
@@ -1,22 +1,58 @@
-# Installation
+# Getting Started
+
+The AdMob module allows you to display adverts to your users. All adverts are served over the Google AdMob network, meaning
+a [Google AdMob account](https://apps.admob.com) is required.
+
+<YouTube id="9qCxo0D-Sak" />
+
+The module supports four types of Ads:
+
+1. Full screen [App Open Ads](/displaying-ads#app-open-ads).
+2. Full screen [Interstitial Ads](/displaying-ads#interstitial-ads).
+3. Full screen [Rewarded Ads](/displaying-ads#rewarded-ads).
+4. Component based [Banner Ads](/displaying-ads#banner-ads).
+
+## Installation
+
+<Tabs groupId="framework" values={[{label: 'React Native', value: 'bare'}, {label: 'Expo', value: 'expo'}]}>
+<TabItem value="bare">
 
 ```bash
 # Install the admob module
-yarn add react-native-google-mobile-ads
+npm install react-native-google-mobile-ads
```

Comment:
Personally I think that newcomers will be more familiar with npm (since it shipps with node) but might be confused by the yarn command, while yarn users immediately know how to adjust the npm command.

I also changed it since the corresponding Expo tab shows an `npx` command which felt more consistent with an `npm` command in the other tab.

I can revert this, but I think the newcomer aspect should be considered. We could also add multiple commands there, but then it would be hard to keep up with all of them (npm, yarn, pnpm and recently bun...).